src/journal/migrations/0027_auto_20190321_1611.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.20 on 2019-03-21 16:11
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('journal', '0026_auto_20190321_1542'),
+    ]
+
+    operations = [
+        migrations.RenameField(
```

Comment:
The changes in these 3 migrations are bit weird, I suppose you changed your mind a couple of times. Should probably have all this changes in journal 0025 instead.

Also, wouldn't this migrations wipe current guest editors data? Shouldn't we migrate those over as IssueEditors?

Reply:
I've added a forwards migration, just need to add backwards.